### PR TITLE
SSE-3132:Created Environment Variables for ServiceNow Parameters and Secrets

### PR DESCRIPTION
**Pull request by DStumpGDS**
Added new Secrets and Parameters to support ServiceNow Integration.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -15,9 +15,14 @@ ZENDESK_TAG='A tag used to route the ticket to the right queue'
 ZENDESK_TAG_ONE_LOGIN_ADMIN_TOOL='A tag used to route the ticket to the Admin tool queue'
 ZENDESK_GROUP_ID='A groupd ID used to assign the ticket to the right group'
 
+# ServiceNow
+SERVICENOW_AUTH_CRDENTIALS='Credentials to access Service Now'
+SERVICENOW_URL='https://onelogingovukdev.service-now.com/api/gds'
```
Comment (BorisP-GDS) [inline .env.example:20]: Remove that URL as this is a secret and replace the value in this file with dummy value.

**Review by BorisP-GDS — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (DStumpGDS) [reply .env.example]: Now changed to https://dummyurl

**Review by BorisP-GDS — APPROVED** [review]
Looks good to me.

File: ci/configure-deployment-parameters.sh [hunk 2]
```diff
@@ -71,6 +75,8 @@ function check-frontend-params {
   check-parameter-set "${parameter}" || write-parameter-value "$parameter" "$([[ $ACCOUNT == production ]] && echo false || echo true)"
   parameter=${PARAMETERS[admin_tool_url]}
   check-parameter-set "${parameter}" || write-parameter-value "$parameter" "https://admin.sign-in.service.gov.uk"
+  parameter=${PARAMETERS[use_stub_servicenow]}
+  check-parameter-set "${parameter}" || write-parameter-value "$parameter" "true"
```
Comment (rafal-surowiec-co) [inline ci/configure-deployment-parameters.sh:79]: check-parameter-set "${parameter}" || write-parameter-value "$parameter" "false"

**Review by rafal-surowiec-co — CHANGES_REQUESTED** [review]

**Review by BorisP-GDS — DISMISSED** [review]
Squash the commits into one.

File: ci/configure-deployment-parameters.sh (hunk 2 above)
Comment (DStumpGDS) [reply ci/configure-deployment-parameters.sh]: Not sure why you have requested this one to be set to False when other such Parameters are set to True but changed as requested.

**Review by BorisP-GDS — APPROVED** [review]
LGTM

**Review by rafal-surowiec-co — APPROVED** [review]

**Review by abdul-gds — APPROVED** [review]
looks good

**Pull request merged**